README.md

```diff
@@ -160,6 +160,40 @@ addServiceWorker({
 ```
 
 
+### Generating a Push Manifest
+
+Polymer-Build can automatically generate a [push manifest](https://github.com/GoogleChrome/http2-push-manifest) for your application. This JSON file can be read by your http2 push-enabled web server to more easily construct the appropriate `Link: <URL>; rel=preload; as=<TYPE>` headers(s) for http2 push/preload.
+
+> **Note:** Push manifests can only be generated for app-shell projects (projects that define a shell and fragments). The manifest describes the following behavior: Requesting the shell should push any shell dependencies as well. Requesting a fragment should push any dependencies of that fragment *that were not already pushed by the shell.*
```

Comment:
why? for an entrypoint-only project, shouldn't we push all the dependencies for the entrypoint?